_data/companies.yml

```
@@ -670,6 +670,13 @@
   events: "?"
   last_update: 2020-03-11
 
+- name: Kentico[[1]](https://www.kentico.com/blog/how-kentico-is-coping-with-covid-19) 
```

```suggestion
- name: "Kentico[[1]](https://www.kentico.com/blog/how-kentico-is-coping-with-covid-19)"
```

Needs to be in quotes to avoid a syntax error.